### DISCUSSION TITLE
Version of the repo vs version of the exe downloaded by sourceforge

### DISCUSSION THREAD

**Discussion by CodeVisio** [winsiderss/systeminformer#2135] [General Discussion]
Hi,

I just discovered this woderful tool.

However the version of the source code downloaded from github is 2.39 while the exe I downloaded from sourceforge has 3.0.7660.

Has the source code been closed somtime in the past?

Thanks

**Comment by Vasilich** [comment]
Where have you downloaded the source code?
In this repository there is latest possible version. 
Version2.39 is the last version of ProcessHacker, but here is the SystemInformer repo.

**Reply by CodeVisio** [reply]
> Where have you downloaded the source code?
> In this repository there is latest possible version.
>Version2.39 is the last version of ProcessHacker, but here is the SystemInformer repo.

The source of my confusion was that under the github page of SystemInformer (here) it is allowed to download another
program (ProcessHacker 2.39 and its source code) that seems a lite version of SystemInformer, while on
[https://systeminformer.sourceforge.io/downloads.php](link) instead you can download SystemInformer 3.0.7660.

Am I wrong?

Thanks

**Comment by dmex** [comment]
> Has the source code been closed somtime in the past?

What happens when you browse https://github.com/processhacker/processhacker or https://github.com/processhacker/processhacker.git ?

> that seems a lite version of SystemInformer

The `lite` version is v2.39 from 2016 while the latest version is v3.1 from 2024

> instead you can download SystemInformer 3.0.7660.

3.0 and 3.1 are canary/nightly builds. 

>  the github page of SystemInformer (here) it is allowed to download 

Only stable versions are uploaded to the download pages. If you execute `git rev-list v2.39..HEAD --count` after checking out the repository there's something like 7886 changes we've been reviewing and preparing for the next release. 

That page will be updated soon :tm:

![image](https://github.com/user-attachments/assets/ae83d8e9-7321-401c-8c08-5b1772aefa24)
